koku/masu/api/trino.py

```diff
@@ -36,6 +36,7 @@ def trino_query(request):
         if query is None:
             errmsg = "Must provide a query key to run."
             return Response({"Error": errmsg}, status=status.HTTP_400_BAD_REQUEST)
+        query = query.removesuffix(";")
```

Comment:
```suggestion
        query = query.strip().removesuffix(";")
```
We could add the `.strip()` here too in case someone adds whitespace at the end of a query, like:
```
"query": "SELECT schema_name FROM information_schema.schemata; "
```